src/catalog/src/builtin.rs

```diff
@@ -2458,16 +2458,36 @@ pub static MZ_SOURCE_STATUS_HISTORY: Lazy<BuiltinSource> = Lazy::new(|| BuiltinS
     sensitivity: DataSensitivity::Public,
 });
 
-pub static MZ_PRIVATELINK_CONNECTION_STATUS_HISTORY: Lazy<BuiltinSource> =
+pub static MZ_AWS_PRIVATELINK_CONNECTION_STATUS_HISTORY: Lazy<BuiltinSource> =
     Lazy::new(|| BuiltinSource {
         name: "mz_aws_privatelink_connection_status_history",
         schema: MZ_INTERNAL_SCHEMA,
         data_source: Some(IntrospectionType::PrivatelinkConnectionStatusHistory),
-        desc: MZ_PRIVATELINK_CONNECTION_STATUS_HISTORY_DESC.clone(),
+        desc: MZ_AWS_PRIVATELINK_CONNECTION_STATUS_HISTORY_DESC.clone(),
         is_retained_metrics_object: false,
         sensitivity: DataSensitivity::Public,
     });
 
+pub const MZ_AWS_PRIVATELINK_CONNECTION_STATUSES: BuiltinView = BuiltinView {
+    name: "mz_aws_privatelink_connection_statuses",
+    schema: MZ_INTERNAL_SCHEMA,
+    column_defs: None,
+    sql: "
+    WITH latest_events AS (
+        SELECT DISTINCT ON(connection_id) occurred_at, connection_id, status
+        FROM mz_internal.mz_sink_status_history
+        ORDER BY connection_id, occurred_at DESC
+    )
+    SELECT
+        conns.id,
+        name,
+        occurred_at as last_status_change_at,
+        status
+    FROM mz_connections AS conns
```

Comment:
I think this needs to be filtered to privatelink connections?

Reply:
Ah, this should have been selecting from the CTE not `mz_connections`. I fixed this but also changed the reporting substantially.